collections/_article/migrating-animations-from-godot-4-0-to-4-3.md

```diff
@@ -0,0 +1,323 @@
+---
+title: "Migrating Animations from Godot 4.0 to 4.3"
+excerpt: "In Godot 4.3, a large number of animation features have been reworked since 4.0, so it's time to migrate your animations."
+categories: ["progress-report"]
```

Comment:
Is `categories: ["progress-report"]` appropriate? In cover image, I write "DEVLOG" same as retarget blog (https://godotengine.org/article/animation-retargeting-in-godot-4-0/), but I can't find that category after Godot web site is renewal.

Reply:
category `progress-report` is right, it is what we use for "devblogs" now. What we add as a caption on the image can be whatever you prefer :)